.storybook/main.js

```diff
@@ -35,7 +35,7 @@ module.exports = {
   ],
   webpackFinal: async (config) => {
```

Comment:
https://61313967cde49b003ae2a860-tymygfwjmc.chromatic.com/?path=/story/atoms-icons-icongrid--default Icons seem to load, so not exactly sure what that `icon-component-loader` file was for, might be remnants of old storybook versions, like the separate webpack config